src/obfl-specification.html

```diff
@@ -2317,6 +2391,22 @@ <h4 class="include" id="L2749">rendering-scenario</h4>
           </dd>
       </dl>
     </dd>
+  <dt>Variables</dt>
+    <dd><dl>
+        <dt>min-block-width</dt>
+          <dd>The variable holding the minimum block with. It can be
+            used in the cost expression, for instance, <code>cost="(- 30
+            $min-block-width)"</code>.</dd>
+        <dt>forced-break-count</dt>
+          <dd>The variable holding the count of forced breaks. It can be
+            used in an expression, for instance, <code>cost="(* 10
```

Comment:
"It can be used in an expression" --> "in a cost expression".

Again the remark here to explain at the top that these variables are for use in the cost attribute, and drop the comment from the individual variables.